src/Gui/CoinRiftWidget.cpp

```diff
@@ -237,7 +243,11 @@ void CoinRiftWidget::resizeGL(int width, int height) {
 
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
-    glOrtho(-1.0, 1.0, -1.0, 1.0, 0.0, 1000.0);
+    SbViewVolume vv;
+    vv.ortho(-1.0, 1.0, -1.0, 1.0, 0.0, 1000.0);
+    SbMatrix affine, proj;
```

Comment:
In general I'd rather not add any new lint if we can help it: can you split this into two lines?